netbout-web/src/main/java/com/netbout/rest/account/TkIndex.java

```diff
@@ -62,7 +63,7 @@ public Response act(final Request req) throws IOException {
         return new RsPage(
             "/xsl/account.xsl",
             this.base,
-            req,
+            new RqWithDefaultHeader(req, "accept", "text/xml"),
```

Comment:
@pinaf @exper0 I would propose `com.google.common.net.HttpHeaders.ACCEPT` 
what do you think? 
we have guava dependence in the project already 

btw i'm with @pinaf on it
according rfc2616 the header starts from upper case